src/components/Input/__tests__/RadioButton.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { renderIntoDocument, cleanup, fireEvent } from "react-testing-library";
+
+import RadioGroup from "../RadioGroup";
+import RadioButton from "../RadioButton";
+
+describe("RadioGroup", () => {
+  afterEach(cleanup);
+
+  it("renders default input", () => {
+    expect(renderGroupComponent()).toMatchSnapshot();
+  });
+
+  it("renders small input", () => {
+    expect(renderGroupComponent({ size: "small" })).toMatchSnapshot();
+  });
+
+  it("renders large input", () => {
+    expect(renderGroupComponent({ size: "large" })).toMatchSnapshot();
+  });
+
+  it("testng on change", () => {
```

Comment:
I think we'll need to add a test that checks only one radio button is "selected" at a time.